src/config.ts

```diff
@@ -5,7 +5,7 @@ import { ListSettings, ListViews, Pagination } from "./types";
 export const IS_TEST = process.env.NODE_ENV === "test";
 export const APP_MOUNT_URI = IS_TEST ? "/" : process.env.APP_MOUNT_URI || "/";
 export const APP_DEFAULT_URI = "/";
-export const API_URI = process.env.API_URI;
+export const API_URI = window.__SALEOR_CONFIG__.API_URI;
```

Comment:
export a function instead of const eg. getApiUrl, then use it everywhere, function allows us to implement some url resolving strategies afterwards